setup.py

```diff
@@ -101,7 +101,7 @@ def is_requirement(line):
 CHANGELOG = open(os.path.join(os.path.dirname(__file__), 'CHANGELOG.rst'), encoding="utf8").read()
 
 setup(
-    name='token-utils',
+    name='edx-token-utils',
```

Comment:
Confirm/question: this is the only place I need to change right? Is it better to change from token-utils > edx-token-utils for the repo and throughout or is it okay to keep as is? Wondering if the name difference could lead to future confusion/issues.

Reply:
using ag "token-utils" doesn't show up a lot so as far as I can see no other places exist to change it

( if you don't have ag or ripgrep install one and change your life :) )